pulumi/package.json

```diff
@@ -10,10 +10,16 @@
   },
   "dependencies": {
     "@shapeshiftoss/cluster-launcher": "^0.19.0",
+    "axios": "^0.27.2",
+    "dotenv": "^16.0.3",
+    "folder-hash": "^4.0.2",
+    "object-hash": "^3.0.0",
```

Comment:
these exist in top level `package.json` and are accessible as hoisted node modules